scripts/broken-links-slack-payload.mjs

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
+import fs from 'fs'
+import core from '@actions/core'
+import { program } from 'commander'
+
+const DEFAULT_INPUT_FILE = 'broken-links-report.json'
+const DEFAULT_OUTPUT_FILE = 'broken-links-slack-payload.json'
+const GROUP_TO_NOTIFY = 'subteam^S010U41C30V|cms-helpdesk' // '<!subteam^S010U41C30V|cms-helpdesk>'
+const RUN_URL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`
+
+// Gather any command line options or provide defaults
+program
+  .version('1.0.0', '-v, --version') /* Not really necessary... */
+  .option('-i, --input-file <input-file>', 'The broken link report to process', DEFAULT_INPUT_FILE)
+  .option('-o, --output-file <output-file>', 'The output location of the Slack payload file', DEFAULT_OUTPUT_FILE)
+  .option('-m, --markdown-url <markdown-url>', 'The URL for the markdown report artifact')
+  .option('-c, --csv-url <csv-url>', 'The URL for the CSV report artifact')
+  .option('-d, --debug', 'Output debug information')
+  .parse(process.argv)
+
+const options = program.opts()
+
+// Wrap up regular and debug logging and only allow if the debug flag is set.
+const say = (level, ...args) => {
```

Comment:
Wrapper around console.log. The intention was to allow console.logs that only show when the debug flag is on. It ends up not very used.